app/src/server/utils/calculateEntryScore.ts

```diff
@@ -0,0 +1,41 @@
+import { type ChatCompletionMessage } from "openai/resources/chat";
+
+// If function names don't match, return 0
+// If function names match and there are no args, return 1
+// If function names match and there are args, return .5 + (num matching args / num args) * .5
```

Comment:
If the fn names match then we should just return the fraction of args in the output that exactly match the fraction of args in the input. Most of the time there's only one possible fn people are calling so you shouldn't get 50% credit for calling it.

Reply:
Works for me